docs/target-browser/README.md

```diff
@@ -64,77 +67,94 @@ kubectl tvk-target-browser get metadata --help
 ```
 
 ## Examples
-  
-  - Get list of backups:
+
+- Get list of backups:
+
   ```bash
   kubectl tvk-target-browser get backup --backup-plan-uid <uid> --target-name <name> --target-namespace <namespace>
   ```
 
- - Get list of backups for backupPlan:
+- Get list of backups using HTTPS:
+  ```bash
+  kubectl tvk-target-browser get backup --backup-plan-uid <uid> --target-name <name> --target-namespace <namespace> --use-https true --certificate-authority <certificate-path>
+  ```
 
+- Get list of backups for backupPlan:
   ```bash
   kubectl tvk-target-browser get backup --backup-plan-uid <uid> --target-name <name> --target-namespace <namespace>
   ```
 
-  - Get specific backup:
+- Get list of backups for backupPlan using HTTPS:
+  ```bash
+  kubectl tvk-target-browser get backup --backup-plan-uid <uid> --target-name <name> --target-namespace <namespace> --use-https true --certificate-authority <certificate-path>
+  ```
+
+- Get specific backup:
   ```bash
   kubectl tvk-target-browser get backup <backup-uid> --target-name <name> --target-namespace <namespace>
   ```
-  - List of backups in Single Namespace:
+- List of backups in Single Namespace:
   ```bash
   kubectl tvk-target-browser get backup --operation-scope SingleNamespace --target-name <name> --target-namespace <namespace>
   ```
-  - List of backups in Multi Namespace/Cluster Scope:
+- List of backups in Multi Namespace/Cluster Scope:
   ```bash
    kubectl tvk-target-browser get backup --operation-scope MultiNamespace --target-name <name> --target-namespace <namespace>
   ```
 
-  - Get list of backupPlans:
+- Get list of backupPlans:
   ```bash
   kubectl tvk-target-browser get backupPlan --target-name <name> --target-namespace <namespace>
   ```
 
-  - Get specific backupPlan:
+- Get specific backupPlan:
   ```bash
   kubectl tvk-target-browser get backupPlan <backup-plan-uid> --target-name <name> --target-namespace <namespace>
   ```
-  - List of backupPlans in Single Namespace:
+- List of backupPlans in Single Namespace:
   ```bash
   kubectl tvk-target-browser get backupPlan --operation-scope SingleNamespace --target-name <name> --target-namespace <namespace>
   ```
-  - List of backupPlans in Multi Namespace/Cluster Scope:
+- List of backupPlans in Multi Namespace/Cluster Scope:
   ```bash
   kubectl tvk-target-browser get backupPlan --operation-scope MultiNamespace --target-name <name> --target-namespace <namespace>
   ```
 
-  - Get metadata of specific backup:
+- Get metadata of specific backup:
   ```bash
   kubectl tvk-target-browser get metadata --backup-uid <uid> --backup-plan-uid <uid> --target-name <name> --target-namespace <namespace>
   ```
 
-  - Get resource metadata of specific backup
+- Get metadata of specific backup using HTTPS:
+  ```bash
+  kubectl tvk-target-browser get metadata --backup-uid <uid> --backup-plan-uid <uid> --target-name <name> --target-namespace <namespace> --use-https true --certificate-authority <certificate-path>
```

Comment:
for boolean flag, no need to pass values `true/false`.. simply flag name is sufficient that will be treated as `true` and if flag is not provided that will be `false`.